resources/lang/de_DE/additional.po

```diff
@@ -107,6 +107,27 @@ msgstr "Account nicht gefunden"
 msgid "oauth.provider-not-found"
 msgstr "OAuth-Provider nicht gefunden"
 
+msgid "oauth.invalid_request"
+msgstr "Der OAuth-Provider lehnt die Anfrage aufgrund eines ungültigen oder fehlenden Parameters ab"
+
+msgid "oauth.unauthorized_client"
+msgstr "Authorisierung als Client beim OAuth-Provider ungültig"
+
+msgid "oauth.access_denied"
+msgstr "Der Nutzer oder der OAuth-Provider hat die Anfrage abgelehnt"
+
+msgid "oauth.unsupported_response_type"
+msgstr "Der OAuth-Provider unterstützt die gewählte Methode zum Erhalt eines Auth-Codes nicht"
+
+msgid "oauth.invalid_scope"
+msgstr "Der angefragte Scope ist ungültig, unbekannt oder ungültig"
```

Comment:
More `ungültig`?

Reply:
Ah, I translated both "invalid" and "malformed" to "invalid". Literally it would be "missgebildet" 😆 but maybe "fehlerhaft" is better.